lib/src/multi_search.dart

```diff
@@ -0,0 +1,24 @@
+import 'api_call.dart';
+
+class MultiSearch {
+  final ApiCall _apiCall;
+  final bool useTextContentType;
+  static const String RESOURCEPATH = '/multi_search';
+
+  const MultiSearch(ApiCall apiCall, {this.useTextContentType = false})
+      : _apiCall = apiCall;
+
+  Future<Map<String, dynamic>> perform(Object searchRequests,
+      {Map<String, String> queryParams}) async {
+    assert(searchRequests is String || searchRequests is Map<String, dynamic>);
+
```

Comment:
@mafreud Is this a good way to ensure that the `searchRequests` would either be a `text/plain` or `application/json`? Because asserts will give runtime error.

Or should I make two different public functions one which accepts `searchRequests` as a `String` and the other which accepts a `Map<String, dynamic>`?